app/views/shared/_save_page_confirmation.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if has_saved_page_confirmation? %>
+  <div class="govuk-!-margin-top-4">
+    <%= render "govuk_publishing_components/components/success_alert", {
+      message: t("components.save_this_page.confirmations.#{personalisation_param}.message"),
+      description: sanitize(additional_text({
+        :additional_class => "govuk-notification-banner__link",
```

Comment:
What's the reason for using this syntax compared with the `additional_class: "govuk-notification-banner__link"` syntax?

Reply:
Not sure why I did it like that to be honest 🙃